devices/tuya.js

```diff
@@ -4279,4 +4279,208 @@ module.exports = [
+        vendor: 'TuYa',
+        description: '4-Gang switch with backlight',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        exposes: [
+            exposes.binary('switch_1', ea.STATE_SET, 'ON', 'OFF').withDescription('Switch 1'),
+            exposes.binary('switch_2', ea.STATE_SET, 'ON', 'OFF').withDescription('Switch 2'),
+            exposes.binary('switch_3', ea.STATE_SET, 'ON', 'OFF').withDescription('Switch 3'),
+            exposes.binary('switch_4', ea.STATE_SET, 'ON', 'OFF').withDescription('Switch 4'),
+            exposes.binary('backlight_mode', ea.STATE_SET, 'ON', 'OFF').withDescription('Backlight Mode'),
+            exposes.binary('child_lock', ea.STATE_SET, 'ON', 'OFF').withDescription('Child Lock'),
+            exposes.enum('relay_status', ea.STATE_SET, ['power_off', 'power_on', 'last']).withDescription('Relay Status'),
+        ],
+        configure: tuya.configureMagicPacket,
+        meta: {
+            tuyaDatapoints: [
+                [1, 'switch_1', tuya.valueConverter.onOffNotStrict],
+                [2, 'switch_2', tuya.valueConverter.onOffNotStrict],
+                [3, 'switch_3', tuya.valueConverter.onOffNotStrict],
+                [4, 'switch_4', tuya.valueConverter.onOffNotStrict],
+                [14, 'relay_status', tuya.valueConverterBasic.lookup({
+                    'power_off': tuya.enum(0),
+                    'power_on': tuya.enum(1),
+                    'last': tuya.enum(2),
+                })],
+                [16, 'backlight_mode', tuya.valueConverter.onOffNotStrict],
+                [101, 'child_lock', tuya.valueConverter.onOffNotStrict],
+            ],
+        },
+    },
+    {
+        fingerprint: [
+            {modelID: 'TS0601', manufacturerName: '_TZE200_p6vz3wzt'},
+        ],
+        model: 'TS0601_cover_4',
+        vendor: 'TuYa',
+        description: 'Curtain / Blind switch',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        onEvent: tuya.onEventSetLocalTime,
+        configure: tuya.configureMagicPacket,
+        meta: {
+            tuyaDatapoints: [
+                [1, 'state', tuya.valueConverterBasic.lookup({'OPEN': tuya.enum(0), 'STOP': tuya.enum(1), 'CLOSE': tuya.enum(2)})],
+                [2, 'position', tuya.valueConverter.raw],
+                [3, 'accurate_calibration', tuya.valueConverterBasic.lookup({'START': 0, 'END': 1})],
+                [7, 'backlight_switch', tuya.valueConverter.onOffNotStrict],
+                [8, 'motor_steering', tuya.valueConverterBasic.lookup({'FORWARD': 0, 'BACKWARD': 1})],
+                [103, 'child_lock', tuya.valueConverter.onOffNotStrict],
+            ],
+        },
+        exposes: [
+            exposes.enum('state', ea.STATE_SET, ['OPEN', 'STOP', 'CLOSE']).withDescription('State'),
+            exposes.numeric('position', ea.STATE_SET).withValueMin(0).withValueMax(100).withDescription('Position'),
```

Comment:
Use `e.cover_position()` instead of a separate `state` and `position` expose

Reply:
make sense